src/sentry/api/endpoints/project_group_index.py

```diff
@@ -691,6 +691,8 @@ def put(self, request, project):
                         ).delete()
                         ignore_until = None
                         result['statusDetails'] = {}
+
+                    issue_ignored.send(project=project, sender=type(self))
```

Comment:
this will cause the signal to be sent for both ignore and un-ignore. I think you want only ignore? (i.e. put it in the `if` clause above)

also @mattrobenolt is this the right pattern or should we be using post_save.connect on GroupSnooze?

based on what matt says, we may have to modify the rest of the code too

Reply:
I think it's fine. It decouples the action of "issue ignored" from GroupSnooze being saved. Then if at some point, our "issue ignored" behavior changes, we just carry the signal around.